addons/stock/static/src/stock_forecasted/forecasted_details.xml

```diff
@@ -23,8 +23,8 @@
                     <td/>
                     <td/>
                 </tr>
-                <tr t-foreach="props.docs.lines" t-as="line" t-key="line.index" t-attf-class="#{line.is_matched and 'o_grid_match'}">
-                    <td t-attf-class="#{line.is_late and 'o_grid_warning'}">
+                <tr t-foreach="props.docs.lines" t-as="line" t-key="line.index" t-attf-class="#{line.is_matched and 'o_grid_match table-light'}">
```

Comment:
I’d remove the custom classes; they seem unnecessary without their relative SCSS.

```suggestion
                <tr t-foreach="props.docs.lines" t-as="line" t-key="line.index" t-attf-class="#{line.is_matched and 'table-light'}">
```

Reply:
> someone could potentially target this class with an xpath ?

I find it very unlikely.
`o_grid_match` "it's not a class" just yet, so it cannot be targeted with `tr[hasclass('o_grid_match')]`, for example.
To target `//tr[last()]` or `//tr[@t-key="line.index"]` would be much more convenient than trying to match the entire  `t-attf-class` attribute 🤔

Anyhow, removing the classes it's not "urgent", these just don't do nothing.
If this change raises eyebrows during tech. review we can keep them in prod. and remove them in the fw port targeting `master`.

Don't resolve this conversation for now, let's let the devs decide ;)